README.md

```diff
@@ -31,11 +31,11 @@ exit
 Install Terraform:
 
 ```bash
-# see https://github.com/hashicorp/terraform/releases
```

Comment:
please undo this hunk. the commented text is used my prefered style of lowercase.